addon/components/rui-spinner/component.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+import layout from './template';
+
+export default Ember.Component.extend({
+  layout: layout,
+  tagName: 'i',
+  classNames: ['fa'],
+  classNameBindings: ['typeComputed'],
+
+  // Defaults
+  type: 'spinner',
+
+  // Constructors
+  classPrefix: 'fa',
+  // NOTE: Supported spinner icons.
+  types: ['circle-o-notch', 'cog', 'gear', 'refresh', 'spinner'],
```

Comment:
Cool. Actually default can be the current size that should fit inline with text elements.  lg should be optional for like page loading, etc.